data/templates/pppoe/peer.tmpl

```diff
@@ -47,8 +47,8 @@ mtu {{ mtu }}
 mru {{ mtu }}
 
 {% if authentication is defined %}
-{{ "user " + authentication.user if authentication.user is defined }}
-{{ "password " + authentication.password if authentication.password is defined }}
+{{ "user \"" + authentication.user + "\"" if authentication.user is defined }}
```

Comment:
Escaping is a bad idea and makes this less readable, please use `'`and `"`, like:
```
{{ 'user "' + authentication.user + '"' if authentication.user is defined }}
```

Reply:
Changed, thanks.